test/Functional/ModuleTest.php

```diff
@@ -60,8 +63,8 @@ public function testProcessParameters(): void
 
         $config = $application->getConfig();
 
-        $this->assertEquals('secret', $config['secret']);
-        $this->assertEquals('parameter', $config['parameter']);
+        $this->assertEquals('secret', $config['secret'] ?? null);
+        $this->assertEquals('parameter', $config['parameter'] ?? null);
```

Comment:
Will we ever have it where both secret and parameter store are null ???

Reply:
These are unit tests so if they are not set then the test will fail - null is an impossible expected value in this test.

In the source code, these things will always be a string:

https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-secretsmanager-2017-10-17.html#getsecretvalue

https://docs.aws.amazon.com/cli/latest/reference/ssm/get-parameters-by-path.html#output